leasing/admin.py

```diff
@@ -602,6 +602,7 @@ class ReceivableTypeAdmin(admin.ModelAdmin):
         "service_unit__name",
         "sap_material_code",
         "sap_order_item_number",
+        "sap_project_number",
```

Comment:
Is there a need to add this also to LeaseTypeAdmin?

Reply:
I'm thinking not really, since neither order_item_number is listed there: https://github.com/City-of-Helsinki/mvj/blob/1184d0350406727ec2b0b3a1ab25ad7322b83344/leasing/admin.py#L358